lib/s3routes/routesUtils.js

```diff
@@ -376,10 +376,13 @@ const routesUtils = {
      * @param {array | undefined} range - range in format of [start, end]
      * if range header contained in request or undefined if not
      * @param {object} log - Werelogs logger
+     * @param {object} [extraParams] - object containing extra params
```

Comment:
please remind me what this is needed for. can we include something in the comment that is generic but also gives an idea of what is going?